Matrix/Source/Matrix3x3.cpp

```diff
@@ -0,0 +1,96 @@
+//
+// Created by Sam Petrov on 18.05.2020.
+//
+
+#include <cassert>
+#include <ctime>
+#include <random>
+#include "Matrix3x3.h"
+
+int Matrix3x3::element(const int i, const int j) const {
+    assert(i < SIZE && j < SIZE);
+    return matrix[i][j];
+}
+
+void Matrix3x3::setElement(const int i, const int j, const int value) {
+    matrix[i][j] = value;
+}
+
+void  Matrix3x3::fillRandomElements(const int minVal, const int maxVal) {
+    std::mt19937 generator(time(0));
+    std::uniform_int_distribution<int> distribution(minVal, maxVal - 1);
+    for(int i = 0; i < SIZE; ++i)
+        for(int j = 0; j < SIZE; ++j)
+            setElement(i, j, distribution(generator));
+}
+
+int Matrix3x3::sumPrincipalDiag() const {
+    int sum = 0;
+    int j = 0;
+    for(int i = 0; i < SIZE; ++i) {
+        sum += matrix[i][j];
+        ++j;
```

Comment:
Я больше не хочу этого видеть!!!
```suggestion
        sum += matrix[i][i];
```
Как это можно развидеть?
Я выложу на http://govnokod.ru/
Везде исправить!